### PR TITLE
i18n(es): Update `framework-components.md`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)
- New or updated content
- Translated content

#### Description

- What does this PR change? Update of `framework-components.md` according to b34822eb26f396fe6e6e2c08822122a0cf295766, 15042052155370e6ffb2fb348d510e424cd02f19 and fbff28449add1335637214032b2f4bcd1de491e7.

### CODE REVIEW

File: src/pages/es/core-concepts/framework-components.md [hunk 1]
```diff
@@ -58,32 +58,35 @@ import MyReactComponent from '../components/MyReactComponent.jsx';
...
 <InteractiveCounter client:visible />
+
+<!-- Este componente no se renderizará en el servidor, pero se renderizará en el 
+cliente cuando la página cargue -->
+<InteractiveModal client:only="svelte" />
 ```
 
 :::caution
-El JavaScript necesario para renderizar el componente del framework (por ejemplo, React, Svelte) se descargará con la página. Las directivas `client:*` solo dictan cuándo se importa el _JavaScript del componente_ y cuándo se hidrata el _componente_.
+El framework de JavaScript (React, Svelte, etc.) necesario para renderear el componente será enviado al navegador con su código JavaScript. Si dos o más componentes en una página usan el mismo framework, el framwork solo se mandará una vez.
```
Comment (kevinzunigacuellar) [inline src/pages/es/core-concepts/framework-components.md:89]:
```suggestion
El framework de JavaScript (React, Svelte, etc.) necesario para renderizar el componente será enviado al navegador junto con el código JavaScript del componente. Si dos o más componentes en una página usan el mismo framework, el framework solo se mandará una vez.
```

File: src/pages/es/core-concepts/framework-components.md [hunk 2]
```diff
@@ -118,6 +121,27 @@ import MyVueComponent from '../components/MyVueComponent.vue';
 Solo los componentes de **Astro** (`.astro`) pueden contener componentes de múltiples frameworks.
 :::
 
+## Pasando props al componente de framework
+
+Tu puedes pasar props desde componentes de Astro a tus componentes de framework:
```
Comment (kevinzunigacuellar) [inline src/pages/es/core-concepts/framework-components.md:126]:
```suggestion
Puedes pasar props desde componentes de Astro a tus componentes de framework:
```

File: src/pages/es/core-concepts/framework-components.md [hunk 3]
```diff
@@ -118,6 +121,27 @@ import MyVueComponent from '../components/MyVueComponent.vue';
...
+import Counter from '../components/Counter.svelte';
+---
+<div>
+  <TodoList initialTodos={["aprender Astro", "revisar PRs"]} />
+  <Counter startingCount={1} />
+</div>
+```
+
+:::caution[pasando funciones como props]
+TU puedes pasar una función como prop a un componente de framework, pero solo si este trabaja con server rendering. Si tu intentas usar la función en un componente hidratado (por ejemplo, como un manejador de evento), un error se producirá.
```
Comment (kevinzunigacuellar) [inline src/pages/es/core-concepts/framework-components.md:140]:
```suggestion
Puedes pasar una función como prop a un componente de framework, pero este solo funcionará al renderizar el componente en el servidor. Si intentas usar la función en un componente hidratado (por ejemplo, como un evento), se producirá un error.
```

**Review by kevinzunigacuellar — COMMENTED** [review]
Deje unos comentarios gracias por la PR @dreyfus92 🎉

**Comment by dreyfus92** [comment]
Gracias por tus comentarios @kevinzunigacuellar  🚀

**Comment by kevinzunigacuellar** [comment]
Let's Get This Merged!

**Pull request merged**